src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericUploadFile.js

```diff
@@ -68,7 +68,7 @@ export const GenericUploadFile = ({
         FileManagementUtils.downloadFile(organizationId, workspaceId, datasetId, setClientFileDescriptorStatus);
       }}
       file={parameter}
-      editMode={context.editMode}
+      editMode={!context.editMode}
```

Comment:
`editMode` doesn't exist anymore so `GenericUploadFile` is always in `editMode` even when scenario is running